### PR TITLE
PR to weather app (project complete)

### PR DESCRIPTION
**Pull request by vichuge**
The application uses an API weather service to find the weather in different parts across the world.

### CODE REVIEW

**Review by MahmoudBakr23 — COMMENTED** [review]
# **INVALID 🛑**

<hr>

Hello there :)
Nice work submitting this project. However, it seems you forgot to set up your Linter. Please do so, then ask for another CRR to get this PR reviewed!
Remember that it's very important to maintain your repo and code professional and ready.

<hr>

Good luck and happy coding 👨‍💻💻

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  if (city === '') {
+    const errors = document.getElementById('error');
+    errors.innerHTML = 'City can\'t be blank';
+    return;
+  }
+  getWeather(city);
+  closeModal();
+};
+
+async function getWeather(city = 'London') {
```
Comment (BrittanyBlake) [inline src/index.js]: - [x] You are asked to use `ES6` syntax, so you should be using `arrow functions` here instead of the `function keyword.`

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @vichuge! 👋 
You have done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] I am having trouble running your app. 
When trying to start your app from the command line I get the following error 👇🏾 
Screen Shot 2021-04-21 at 5 20 34 PM

When I go live from the index.html file, I am able to open the app but nothing happens after I click **Find Weather** 👇🏾 
Screen Recording 2021-04-21 at 05 23 57 PM




****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,62 @@
...
+### Setup
+
+- Install [git](https://git-scm.com/downloads)
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `git clone https://github.com/vichuge/weather-app.git`
+- open the folder
+- double click index.html
+- Here comes the website on the local machine.
```
Comment (teekaytech) [inline README.md:29]: The setup instruction in the Readme is not complete: you should add instructions on how the user can build for deployment, run a local server, etc.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,62 @@
...
-The application uses an API weather service to find the info to different parts across the world.
+<h3 align="center">Weather app - Javascript</h3>
+
+
+<p align="center">Weather app in JavaScript is a practice project, which includes the API service to see the weather to more than 200,000 cities around the world. We used Javascript, HTML5 and CSS3, to implement it.</p>
+
+![screenshot](/screenshots/Screenshot_1.png)
+
+## Built With
+
```
Comment (teekaytech) [inline README.md:9]: Kindly deploy your application and add the live demo link to the Readme.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @vichuge :wave:

__I must commend your efforts so far in this project. However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] As mentioned by the last reviewer, running `npm start` still leads to the error mentioned previously. I believe this is so because the `webpack-dev-server` package is missing in your `devDependencies`. Kindly fix it.


**[ OPTIONAL CHANGES ]**

- [ ] Kindly add a toggle control to determine if the result will be shown in Celsius or Fahrenheit. 
- [ ] You should also make the page change its appearance based on the received data

_Note that these are supposed to be mandatory requirements. I only added it as optional because the last reviewer did not mention it. Therefore, it is strongly advised that you make the changes._

* * *
**Please make the required changes and submit another review request when you are done.**

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @vichuge :tada: :clinking_glasses: 

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**Comment(s)**
- I mentioned in my last review, a couple of features to be implemented. It will be nice if you can consider adding these features before you move onto the next project. This is because those features are part of the project. I did not enforce them because the first reviewer did not mention them.

Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**